<?xml version="1.0"?>
<launch>

  <node name="spg_tts" pkg="tta_blb" type="node_TTS.py" respawn="False" output="log" /> 
</launch>